### ISSUE TITLE
Font - body

### ISSUE THREAD

**Issue by Alex-e107nl** [Jimako-e107-themes/aeolus#1]
Hi, i have changed my version of the aeolus theme so it reflects your latest changes... there's one thing that i think has gone wrong, 
in the style,css the font of the body is not defined right... (i think)

It is now:

body {
  font-family: var(--font-family);
  color: var(--black);
  font-size: 16px;
  position: relative;
}

But with this it looks like before...

body {
  font-family: var(--font-sans-serif);
  color: var(--black);
  font-size: 16px;
  position: relative;
}

style.css - line 6036

Thx for all your work!

**Comment by Alex-e107nl** [comment]
Sorry it's late... forget it for now :-)

**Comment by Alex-e107nl** [comment]
Sorry it was the disappearing of the Inter font that was the problem, for this theme it's for me the best to work with... so i have brought it back :-)

**Comment by Jimmi08** [comment]
@Alex-e107nl there was change in theme.xml - loading full bootstrap instead using custom bootstrap version, because not solved issues related to this way. So there has to be different approach now.   

Intel wasn't loaded because commenting its loading here: 
register_fonts() in theme.php

@import can't be used because then e107 css caching fails 

Oh, and do you know that you should put all your css changes into file style_custom.scss

For sure, I will be reorganized css files and other stuff.

**Comment by Alex-e107nl** [comment]
Oke, i just try to keep up with your git of aeolus... Inter font activated in theme.php and the @import in style.css removed.. looks oke! Where must the file style_custom.css be placed, root theme? Just playing with it....

**Comment by Jimmi08** [comment]
Yes, it is last loaded css by core (after style.css) and because it is not in pack, it is not overriden when you update core. It was mainly used if you use directly bootstrap3 or bootstrap5 class. 

In the best scenario (when I finish) you should place there only new css variables...

**Comment by Alex-e107nl** [comment]
Oke, just one question, why did you choose for the pages navigation and not the Home? I have to test it further but on my mobile the pages menu does not work, when there are to much links you cannot scroll down... And i liked the Home version much much more :-) Maybe it's because of me playing with the css files...

**Comment by Jimmi08** [comment]
I don't remember. Do you mean this one
![image](https://user-images.githubusercontent.com/5429548/194763168-9ad4cfc7-9cfa-40c8-9cc9-8d1ff867f59e.png)

Instead of 
![image](https://user-images.githubusercontent.com/5429548/194763198-50f90618-9583-4f29-8b23-0bd596fec27b.png)

I probably messed something, because live demo has latest one and my local copy that first one. Yes, I played with it because you mentioned it somehow, but I really don't remember. Maybe just wrong css is included.  

Do you mind to use plugin for this?

**Comment by Alex-e107nl** [comment]
Hihi i the home navigation is the second one, can't it be in the theme prefs?
